a = (1, 2, 3, 4)
print(a)

# 오류 발생
a[2] = 7
